_data/authors.yaml

```diff
@@ -144,7 +144,7 @@ cesarsaavedra:
   emailhash: "33c2cb775850e595c2dc1dedd88ef98d"
   job_title: ""
   twitter: "cesar_saavedr"
-  bio: "Cesar Saavedra focuses on Eclipse MicroProfile, OpenJDK, Jakarta EE, and Quarkus. He has been a technical seller, consultant, and developer and now enjoys doing technical product marketing for Red Hat Middleware. In addition, he is a MicroProfile committer, has authored a book, white papers, blog posts, and participated in conferences as a speaker and delivered webinars to end customers, partners and internal audiences. He has over 28 years of IT experience, many of them in the application server market, hybrid integration, API management and Process Automation. Before Red Hat, Cesar was at IBM, Oracle, BEA Systems, TIBCO, Lucent Technologies, and AT&T Bell Laboratories. In his spare time, Cesar enjoys spending time with his family, traveling and playing soccer." 
+  bio: "Cesar Saavedra focuses on Eclipse MicroProfile, OpenJDK, Jakarta EE, and Quarkus. He has been a technical seller, consultant, and developer and now enjoys doing technical product marketing for Red Hat Middleware. In addition, he is a MicroProfile committer, has authored a book, white papers, blog posts, and participated in conferences as a speaker and delivered webinars to end customers, partners and internal audiences. He has over 28 years of IT experience, many of them in the application server market, hybrid integration, API management and Process Automation. Before Red Hat, Cesar was at IBM, Oracle, BEA Systems, TIBCO, Lucent Technologies, and AT&T Bell Laboratories. In his spare time, Cesar enjoys spending time with his family, traveling and playing soccer."
```

Comment:
Weird there is a diff here. do you know why @phillip-kruger

Reply:
??? W.t.f ? Hold on I am looking.